package.json

```diff
@@ -9,6 +9,9 @@
     "packages/*",
     "packages/todo-example/*"
   ],
+  "dependencies": {
+    "tslib": "^2.4.0"
+  },
```

Comment:
Uhm I think you actually need to add it to the individual package workspaces within this repository instead of the mono-repository root